modules/social.py

```diff
@@ -1043,6 +1098,71 @@ async def _profile_validate_allow(self, ctx, member: tools.ResolveUser):
         self.validate_allowed_users.append(member.id)
         return await ctx.message.reply(f'{config.greenTick} {member} temporarily added to allowlist')
 
+    @commands.has_any_role(config.moderator, config.eh)
+    @commands.group(name='trivia')
+    async def _trivia(self, ctx):
+        return
+
+    @commands.has_any_role(config.moderator, config.eh)
+    @_trivia.command(name='award')
+    async def _trivia_award(self, ctx, members: commands.Greedy[tools.ResolveUser]):
+        '''Increase the trivia award trophy by one tier for one or more users'''
+        stats = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0, 8: 0, 9: 0}
+        failed = []
+        msg = await ctx.send(f'{config.loading} Processing awards to {len(members)} member(s)...')
+        for m in members:
+            try:
+                newLevel = await self.modify_trivia_level(m)
+                stats[newLevel] += 1
+
+            except IndexError:
+                failed.append(f'{m.mention} ({m.id})')
+
+        embed = discord.Embed(title='Command Completion Stats')
+
+        successList = [(key, value) for key, value in stats.items() if value != 0]
+        embed.description = f'Trivia awards granted to **{len(members) - len(failed)}**.{" List of trophies user(s) now have:" if successList else ""}\n\n'
+        for item in successList:
```

Comment:
Replace `for item in successList` with `for (index, count) in successList` which allows:
`item[0]` → `index`
`item[1]` → `count`